addons/im_livechat/static/src/models/messaging.js

```diff
@@ -14,3 +18,25 @@ addFields('Messaging', {
         readonly: true,
     }),
 });
+
+patchRecordMethods('Messaging', {
+     /**
+     * @override
+     */
+    async performInitRpc() {
+        if (this.isInPublicLivechat) {
+            return {};
+        } else {
+            return this._super();
+        }
+    },
+    /**
+     * @override
+     */
+    _computeNotificationHandler() {
+        if (this.isInPublicLivechat) {
+            return clear();
+        }
```

Comment:
strange to have to kill it, as handling notifications would be desirable in livechat too